modules/core/src/viewports/viewport.js

```diff
@@ -70,6 +70,7 @@ export default class Viewport {
     // Silently allow apps to send in w,h = 0,0
     this.width = width || 1;
     this.height = height || 1;
+    this.frustumPlanes = {};
```

Comment:
Nit: Since there is no underscore, it looks like a public prop. But it requires going through the getFrustumPlanes function....